app/models/ama.rb

```diff
@@ -0,0 +1,62 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: amas
+#
+#  id                    :integer          not null, primary key
+#  ama_subscribers_count :integer          default(0), not null
+#  description           :string           not null
+#  end_date              :datetime         not null
+#  start_date            :datetime         not null
```

Comment:
Shouldn't this be nullable? As I understand we'll want to be able to create AMAs without dates and original posts, so that they can be added later when AMAs are scheduled.
```
- Upcoming AMAs (Shown as Current Month, Next Month, The Following Month and More Upcoming)
```